daml/splice-amulet/daml/Splice/AmuletConfig.daml

```diff
@@ -45,6 +45,7 @@ data AmuletConfig unit = AmuletConfig with
       -- that should be used for command submissions.
     transferPreapprovalFee : Optional Decimal -- ^ Fee for keeping a transfer pre-approval around.
     featuredAppActivityMarkerAmount : Optional Decimal -- ^ $-amount used for the conversion from FeaturedAppActivityMarker -> AppRewardCoupon
+    developmentFundConfig : Optional DevelopmentFundConfig -- ^ Configuration defining the beneficiary and allocation rate for the Development Fund.
```

Comment:
```suggestion
    developmentFundConfig : Optional DevelopmentFundConfig 
      -- ^ Configuration for the development fund mechanics from CIP-0082.
```

avoid repeating record fields in the comments referencing these records. These comments go stale quite easily.